README.md

```diff
@@ -6,59 +6,119 @@ DUS leverages the power of Amazon Textract, Amazon Comprehend , Amazon Comprehen
 
 ## Architecture Diagram
 
-
 ![img](./images/DUS_Arch.png)
 
+Current document formats supported : **PDF,JPG,PNG**
 
+## 2. CICD Deploy
 
-Current document formats supported : **PDF,JPG,PNG**
+### Requirements
 
+- aws cli
 
-##  Development Deploy
+  `sudo yum -y install aws-cli`
 
-The instructions below cover installation on Unix-based Operating systems like macOS and Linux. 
-You can use a AWS Cloud9 enviornment or EC2 instance (recommended: t3.large or higher on Amazon Linux platform) to deploy the solution
+- pip3 (Required to install packages)
+
+  `curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && python3 get-pip.py`
+
+### Getting Started with Full Deploy
+
+- Create a bucket to act as the target Amazon S3 distribution bucket
+
+_Note:_ You will have to create an S3 bucket with the template 'my-bucket-name-<aws_region>'; aws_region is where you are testing the customized solution.
+
+For example, you create a bucket called `my-solutions-bucket-us-east-1`,
+
+- Now build the distributable:
+
+```
+chmod +x ./deployment/build-s3-dist.sh
+./deployment/build-s3-dist.sh <bucket-name-minus-region> <solution-name> <version>
+```
+
+For example,
+
+```
+./deployment/build-s3-dist.sh my-solutions-bucket document-understanding-solution v1.0.0
+```
+
+- Deploy the distributable to an Amazon S3 bucket in your account. _Note:_ you must have the AWS Command Line Interface installed.
+
+```
+aws s3 cp ./deployment/global-s3-assets/ s3://my-bucket-name-<aws_region>/<solution_name>/<my-version>/ --recursive --acl bucket-owner-full-control --profile aws-cred-profile-name
+aws s3 cp ./deployment/regional-s3-assets/ s3://my-bucket-name-<aws_region>/<solution_name>/<my-version>/ --recursive --acl bucket-owner-full-control --profile aws-cred-profile-name
+```
+
+- Get the link of the document-understanding-solution.template uploaded to your Amazon S3 bucket.
+- Deploy the Document Understanding solution to your account by launching a new AWS CloudFormation stack using the link of the document-understanding-solution.template.
+
+```
+aws cloudformation create-stack --stack-name DocumentUnderstandingSolutionCICD --template-url https://my-bucket-name-<aws_region>.s3.amazonaws.com/<solution_name>/<my_version>/document-understanding-solution.template --parameters ParameterKey=Email,ParameterValue=<my_email> --capabilities CAPABILITY_NAMED_IAM --disable-rollback
```

Comment:
Could you please also mention on how we can enable the Read Only (RO) mode or the Kendra enabled mode?

Reply:
I think it calls yarn and yarn deploy so the steps are the same: set respective flags to true.

Reply:
> I think it calls yarn and yarn deploy so the steps are the same: set respective flags to true.

CICD would be eventually used as a 1click deploy via Cloudforamtion. In that situration, we cannot expect the customers to change the package.json file as that wont be a viable option. Probably passing them as parameters in the master CICD cfn template is the ideal approach

Reply:
Updated to hardcode ROmode=False and enableKendra=True

Reply:
Could we pass them as CloudFormation parameters? What if folks using the CICD mode (on AWS  Solutions) dont want Kendra or want the RO mode?
Ideally, both CICD and Dev deploy should have (near to )same flexibility
I was wondering if we can add them as parameters in document-understanding-solution.template and update the buildspec [here](https://github.com/awslabs/document-understanding-solution/blob/add-vpc/deployment/document-understanding-solution.template#L192)

Reply:
Could you please mention how to enable/disable the ro , kendra-enabled modes when using the CICD deployment and using ```aws cloudformation deploy``` by passing parameters

Reply:
updated README